plugins/devlink/README.md

```diff
@@ -22,9 +22,10 @@ This plugin supports the following protocol ID:
 
 ## GUID Generation
 
-These devices use the standard USB DeviceInstanceId values, e.g.
+These devices use custom instance IDs consisting of the component name and the device PSID.
```

Comment:
PSID is something mlx5 specific. Each driver has a different set of exposed "fixed versions". Therefore my goal was to make it as generic as possible to cope with all possible drivers without need to do any changes in fwupd.

Reply:
I don't think that makes for a good API surface -- adding an extra nlattr would change the instance ID and thus the GUID. People using fwupd must be able to match GUIDs now and 10 years into the future, and so I think it makes a lot of sense documenting *exactly* what instance IDs are being constructed.